package.json

```diff
@@ -35,21 +35,21 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "pino": "^8.15.6",
-    "undici": "^5.25.4",
-    "undici-retry": "^2.1.0"
+    "pino": "^8.16.1",
+    "undici": "^5.27.0",
+    "undici-retry": "^3.1.2"
```

Comment:
In case you are curious about how this was implemented on undici-retry side: https://github.com/kibertoad/undici-retry/pull/22/files